cmd/datamon/cmd/context.go

```diff
@@ -28,10 +31,24 @@ var contextTemplate *template.Template
 
 func init() {
 	rootCmd.AddCommand(ContextCmd)
-	addContextFlag(ContextCmd)
 
 	contextTemplate = func() *template.Template {
-		const listLineTemplateString = `{{.Version}} , {{.Name}} , {{.WAL}} , {{.ReadLog}} , {{.Blob}} , {{.Metadata}} , {{.VMetadata}}`
+		const listLineTemplateString = `Model Version: {{.Version}}
```

Comment:
everywhere else datamon needs to print metadata to stdout, it uses the ` , `-delimited format.

i suggest we keep a consistent user interface at the cli:  all `get` and `list` commands should print the same kind of output.

now, we have had requests to change the output format and/or allow different formats for this kind of output (e.g. Gabe has mentioned wanting to use json).

my only request here is for consistency:  let's have all the `get` and `list` commands do about the same thing if possible.

Reply:
ok will do that. I just found out too that the output was difficult to read...

Reply:
ok done @ransomw1c